airflow-core/src/airflow/ui/src/pages/Pools/Pools.tsx

```diff
@@ -60,11 +71,16 @@ export const Pools = () => {
         <AddPoolButton />
       </HStack>
       <Box mt={4}>
-        {isLoading ? (
-          <Skeleton height="100px" />
-        ) : (
-          data?.pools.map((pool) => <PoolBar key={pool.name} pool={pool} />)
-        )}
+        <DataTable
```

Comment:
Since we don't render this as a regular table but more like a list of cards. I would follow what we do in for [AssetEvents](https://github.com/apache/airflow/blob/main/airflow-core/src/airflow/ui/src/components/Assets/AssetEvents.tsx). 

We create a `cardDef` that accepts a `PoolResponse` and renders `<PoolBar />` or `<Skeleton />`. We then pass that cardDef to the DataTable here.

Reply:
okay. made the requested changes using the asset events structure.